Cartfile

```diff
@@ -0,0 +1,3 @@
+github "dhardiman/Fetch"
+github "owensd/json-swift" "master"
```

Comment:
Last tagged version is behind master, and does not compile due to latest Swift syntax.